bug(error): errors.As: *target must be interface or implement error, found *starlark.EvalError

## Description

The error handling fails if there is an eval error:

```
------
panic: errors.As: *target must be interface or implement error, found *starlark.EvalError

goroutine 1 [running]:
github.com/cockroachdb/errors/errutil.As({0x1291cc0, 0xc0001e8810}, {0xfb2ac0, 0xc00003a740})
        github.com/cockroachdb/errors@v1.9.0/errutil/as.go:49 +0x47a
github.com/cockroachdb/errors.As(...)
        github.com/cockroachdb/errors@v1.9.0/errutil_api.go:194
main.handleErr({0x1291cc0, 0xc0001e8810})
        github.com/tensorchord/envd/cmd/envd/main.go:53 +0x153
main.main()
        github.com/tensorchord/envd/cmd/envd/main.go:65 +0x33
```

## Reproduction

## Additional Info

---
<!-- Issue Author: Don't delete this message to encourage other users to support your issue! -->
**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

--- a/cmd/envd/main.go
+++ b/cmd/envd/main.go
@@ -48,8 +48,8 @@ func handleErr(err error) {
 	}
 
 	rootCause := errors.Cause(err)
-	var evalErr starlark.EvalError
-	var syntaxErr syntax.Error
+	var evalErr *starlark.EvalError
+	var syntaxErr *syntax.Error
 	if ok := errors.As(err, &evalErr); ok {
 		fmt.Fprintln(os.Stderr, evalErr.Backtrace())
 	} else if ok := errors.As(err, &syntaxErr); ok {

